mtp_api/settings/base.py

```diff
@@ -387,3 +387,7 @@
     from .local import *  # noqa
 except ImportError:
     pass
+
+
+# TODO: remove after upgrade to django 3.2
+DEFAULT_HASHING_ALGORITHM = 'sha1'
```

Comment:
How is this used? Does it have an impact on things which are persisted in the DB and make moving to a different hashing method problematic? (e.g. users passwords or something like that)

Reply:
wherever a piece of data is signed (e.g. session cookie), it used to use sha1. the future default is sha256, the docs suggest using this setting _during_ migration if there are both old and new djangos running concurrently (which will be the case for ~5min)

Reply:
Ah OK, so it sounds like it's only used for sessions which means worst case scenario user is logged out/needs to log back in 👍